common/migrations/db/m171018_141344_alterMySQLSchemaToBeMB4.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
I think we should create database with utf8mb4 charset, not change it later. You can describe  in docs queries to convert already created tables

Reply:
ADDED logic to append/update .env value during migrations as well as set ./common/config to use .env to select DB charset.